game/scripts/npc/items/item_shivas_cuirass.txt

```diff
@@ -115,23 +115,23 @@
 			"10"
 			{
 				"var_type"				"FIELD_INTEGER"
-				"aura_attack_speed"		"-55"
+				"aura_attack_speed"		"-85 -95"
 			}
 			// Assault Cuirass Parameters
 			"11"
 			{
 				"var_type"				"FIELD_INTEGER"
-				"aura_negative_armor"	"-5"
```

Comment:
I have a post in balance-discussion regarding -armor values. The ones we have for every item but Crest are way undertoned. I'll be fixing this soon.